lib/scaleway/bootstrap/master-cluster.tf

```diff
@@ -0,0 +1,41 @@
+resource "scaleway_k8s_cluster" "kubernetes_cluster" {
+  name    = "qovery-${var.kubernetes_cluster_id}"
+  version = "1.21.1"
```

Comment:
1. please use 1.20.x
2. Like DO, user should only set the majo+minor version, we handle the rest (update). Be sure we can set only major+minor, otherwise you'll have to handle it in the engine by calling scaleway API

Reply:
Unfortunately, Scaleway doesn't seem to support it for the time being, I will ask the question though.

Reply:
Actually they do support it, that's great :)